### PR TITLE
Adding polya gamma moment and corresponding tests

### PR DESCRIPTION
**Pull request by jlopezarriaza**
Adding polya gamma moment and corresponding tests

### CODE REVIEW

**Comment by jlopezarriaza** [comment]
I'm not 100% what's causing the checks to fail

**Comment by ricardoV94** [comment]
> I'm not 100% what's causing the checks to fail

#5191

File: pymc/tests/test_distributions_moments.py [hunk 1]
```diff
@@ -898,3 +899,39 @@ def test_rice_moment(nu, sigma, size, expected):
...
+                0.25,
+                0.46211715726000974,
+                0.5711956169668236,
+                0.6034321690965776,
+                0.6025172375473855,
+            ),
+        ),
+    ],
+)
+def test_polyagamma_moment(h, z, size, expected):
```
Comment (ricardoV94) [inline pymc/tests/test_distributions_moments.py:1035]: Can you add a test where size is not None?

**Review by ricardoV94 — CHANGES_REQUESTED** [review]

File: pymc/tests/test_distributions_moments.py (hunk 1 above)
Comment (jlopezarriaza) [reply pymc/tests/test_distributions_moments.py:1035]: Done, overlooked that.

Comment (ricardoV94) [reply pymc/tests/test_distributions_moments.py:1035]: No problem and thanks!

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
Thanks @jlopezarriaza
